ansible/roles/nginx/templates/profile.j2

```diff
@@ -0,0 +1,20 @@
+server {
+    listen  80;
+    server_name {{ server.name }};
+
+    root /var/www/{{ server.name }}/web;
+
+    location / {
+        try_files $uri /app.php$is_args$args;
```

Comment:
since this is dev only, this server config can be adjusted to point to `app_dev.php` exclusively. Makes our lives easier as then we don't have to include in the URLs, which coincidentally helps with SAML :wink: